lib/libutee/include/user_ta_header.h

```diff
@@ -32,7 +32,16 @@
 #define TA_FLAG_DEVICE_ENUM		(1 << 9)  /* without tee-supplicant */
 #define TA_FLAG_DEVICE_ENUM_SUPP	(1 << 10) /* with tee-supplicant */
 
-#define TA_FLAGS_MASK			GENMASK_32(10, 0)
+	/*
+	 * The flag is to determine if TA get panic when an unsupported
+	 * function ID is called. If this flag is set, it returns an error
```

Comment:
".. _entry_ function ID .."?

Reply:
yeah, "entry function ID" will be more clear.